routing/base/astar_algorithm.hpp

```diff
@@ -13,18 +13,43 @@
 namespace routing
 {
 
-template <typename TVertexType>
+template <typename TWeightType>
```

Comment:
s/TWeightType/WeightType/

Big-T before types is an obsolete notation, not needed now.